doc/specs/vulkan/chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -347,6 +347,8 @@ ifdef::VK_KHX_device_group[]
 endif::VK_KHX_device_group[]
   * [[VUID-VkSwapchainCreateInfoKHR-oldSwapchain-01674]]
     pname:oldSwapchain must: not be in the retired state
+  * If pname:oldSwapchain is not dlink:VK_NULL_HANDLE, pname:surface must:
```

Comment:
This should probably be more complete, echoing the language describing this field: "pname:oldSwapchain must: be dlink:VK_NULL_HANDLE or the non-retired swapchain associated with pname:surface."

Reply:
Well, yes. That is what the VU above this one says.
It would be a matter of preference, whether they should be two separate VUs or one.

From validation layers standpoint, these seemed like two different things probably checked in two different ways.